cernopendata/modules/fixtures/data/docs/delphi-guide-docker/delphi-guide-docker.md

```diff
@@ -0,0 +1,48 @@
+# Running DELPHI software from a container
+An alternative way to run DELPHI software is to use a container image. The images are automatically created from a CI when the software stack is being build, and contain the full software stack.
+
+## Available images
+Images are available in the registry of the deployment project at CERNs gitlab instance, https://gitlab.cern.ch/delphi/deployment/container_registry
+
+In this guide we will use the latest Alma9 image.
+
+## Starting the container
+In this guide, we will assume to be running on a Linux desktop, which has podman and podman-docker installed. Some of the applications require a graphical interface, namely the CERNLIB applications paw, paw++ and kxterm, as well as the event display.
```

Comment:
Interesting that the guide title speaks of using Docker, but the guide body only shows Podman :wink: 

Perhaps you could make it more explicit with respect to container technologies? For example:

An alternative way to run DELPHI software is to use a container image.  This guide assumes that you are using Podman (link), but you can also use other popular container technologies such as Docker.


(and making a link to https://podman.io/ because it may not be so well known by lambda readers

Reply:
Well spotted. I have changed the phrasing accordingly.